ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/OpenApiServerDcatController.java

```diff
@@ -0,0 +1,91 @@
+									@ExampleObject(value = """
+											_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/dcat#Catalog> .
+											_:genid1 <http://purl.org/dc/terms/title> "My LDES'es"@en .
+											_:genid1 <http://purl.org/dc/terms/description> "All LDES'es from publiser X"@en .
+											""")
+							}),
+							@Content(mediaType = contentTypeJSONLD, schema = @Schema(implementation = String.class), examples = {
+									@ExampleObject(value = """
+											[{"@type":["http://www.w3.org/ns/dcat#Catalog"],
+											"http://purl.org/dc/terms/title":[{"@value":"My LDES'es","@language":"en"}],
+											"http://purl.org/dc/terms/description":[{"@value":"All LDES'es from publiser X","@language":"en"}]},{"@id":"http://www.w3.org/ns/dcat#Catalog"}]
+											""")
+							})
+					}) Model dcat);
+
+	@ApiResponse(responseCode = "200")
+	@ApiResponse(responseCode = "400")
+	@ApiResponse(responseCode = "404")
+	@Operation(summary = "Update DCAT configuration for the server")
+	void putServerDcat(
+			@Parameter(name = "catalogId", content = @Content(schema = @Schema(implementation = String.class), examples = @ExampleObject(value = "e1c9443a-ab9f-407f-a65b-09d69f481966"))) String catalogId,
+			@RequestBody(description = "A blank node of type dcat:Catalog with only dcat:Catalog properties " +
+					"(e.g. dct:title) and relations (e.g. dct:publisher), excluding relations " +
+					"(non-configurable metadata) to dcat:Dataset and dcat:DataService", content = {
+							@Content(mediaType = contentTypeTurtle, schema = @Schema(implementation = Model.class), examples = {
+									@ExampleObject(value = """
+											@prefix dct:   <http://purl.org/dc/terms/> .
+											@prefix dcat:  <http://www.w3.org/ns/dcat#> .
+
+											[] a dcat:Catalog ;
+											  dct:title "My LDES'es"@en ;
+											  dct:description "All LDES'es from publiser X"@en .
+											""")
+							}),
+							@Content(mediaType = contentTypeNQuads, schema = @Schema(implementation = String.class), examples = {
+									@ExampleObject(value = """
+											_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/dcat#Catalog> .
+											_:genid1 <http://purl.org/dc/terms/title> "My LDES'es"@en .
+											_:genid1 <http://purl.org/dc/terms/description> "All LDES'es from publiser X"@en .
+											""")
+							}),
+							@Content(mediaType = contentTypeJSONLD, schema = @Schema(implementation = String.class), examples = {
+									@ExampleObject(value = """
+											[{"@type":["http://www.w3.org/ns/dcat#Catalog"],
+											"http://purl.org/dc/terms/title":[{"@value":"My LDES'es","@language":"en"}],
+											"http://purl.org/dc/terms/description":[{"@value":"All LDES'es from publiser X","@language":"en"}]},{"@id":"http://www.w3.org/ns/dcat#Catalog"}]
+											""")
+							})
+					}) Model dcat);
+
+	@ApiResponse(responseCode = "200")
+	@ApiResponse(responseCode = "404")
```

Comment:
I would not return a 404 on a DELETE as it does not makes a lot of sense. Delete should also be idempotent: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE

Just delete it if it is present and don't do anything if it isn't. The person calling this endpoint just wants this resource to not exist anymore.